### ISSUE TITLE
New name for the extension

### ISSUE THREAD

**Issue by OhMyGuus** [OhMyGuus/I-Dont-Care-About-Cookies#17]
We need a new name for the extension so people aren't getting confused (and so we can upload it).
There are already some nice ideas from the other post but I still need a bit more input so we can decide the best name 

1. I don't care about Cookies debloated
2. I don't care about Cookies or Avast 
3. _Cookie? No, thank you!_ (At the moment it hides or accepts the cookies so misleading at the moment)
4. _Cookie rejecter_ (At the moment it hides or accepts the cookies so misleading at the moment)
5. Cookie popup hider

**Moved it to a poll using discussions :**   [New name for the extension #23](https://github.com/OhMyGuus/I-Dont-Care-About-Cookies/discussions/23)

**Comment by pineapples5972** [comment]
Here is the cool one - "I still don't care about cookies."

**Comment by negulescus** [comment]
- I'm full, thank you
- Cookie monster
- Broken Jar

**Comment by andrigamerita** [comment]
I also suggest "I don't give a damn about cookies"

**Comment by bwims** [comment]
TL;DR   I **literally** don't care about cookies - hence "I don't give a damn about cookies" is +1

I know some people are upset at the thought of accepting cookies.   I have to admit that my annoyance at data gathering is less than at the dishonest implementation of the EU requirement, in that there is a button to press to accept, or (usually a button to "manage" which cookies to accept or some such.  

Sometimes (rarely) you get accept or reject, but I think the author probably didn't want to get into multiple button pushes and parsing the different ways to "manage" cookies, and I presume uses the reject button if available or accept otherwise.

If so, it mirrors my work practice, which is rather than manually "manage" my cookies, I reject if there is a button for it or accept if that's the one-button choice.

In that instance, I **literally** don't care about cookies.

In my opinion, that's probably why the extension is named that way.

I'd rather have something that works (which doesn't send my details to Avast) than something that fails, in attempting the near-impossible - and very grateful for it, I might add.

"I don't give a damn about cookies" is +1

**Comment by zer0fl0w** [comment]
cookie om nom nom

**Comment by flxzt** [comment]
What about `Cookie Eater`?

**Comment by HolgerAusB** [comment]
Don't ask me about cookies

**Comment by frauhottelmann** [comment]
Cookies be gone

**Comment by HolgerAusB** [comment]
"Where have all the cookies gone"     ... long time passing (Original: 'Where Have All the Flowers Gone', song of Peter, Paul and Mary)

**Comment by Gaurav-Malviya** [comment]
Anti-Cookie

**Comment by OhMyGuus** [comment]
**Thanks for all the ideas, I put all the options that I liked the most into a poll:**   [New name for the extension #23](https://github.com/OhMyGuus/I-Dont-Care-About-Cookies/discussions/23)

**Comment by OhMyGuus** [comment]
Closed since we decided on the name
